src/components/DeleteWallet.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <Card
+    class="card"
+    :title="$t('delete_wallet')"
+    :border="true"
+    type="danger"
+    shadow="thin"
+  >
+    <p class="text">
+      <span class="bold">{{ $t('please_note').toUpperCase() }}</span>
+      {{ $t('delete_wallet_info') }}
+    </p>
+    <div class="btn-container">
```

Comment:
I don't find consistent the buttons on this section. We have four buttons with three different styles: text, icon + text, icon. Personally I'd use icon + text or only text
![image](https://user-images.githubusercontent.com/24905681/129555764-0be2842c-9011-4893-8e02-d122fbb47476.png)
![image](https://user-images.githubusercontent.com/24905681/129555792-6e894b1f-6d63-489f-88fc-cc9302f72457.png)
![image](https://user-images.githubusercontent.com/24905681/129555808-91364d94-d9f3-4377-a337-48d6b2f6a8c3.png)
![image](https://user-images.githubusercontent.com/24905681/129555822-2efc201a-142e-470b-ab68-f7a4ddfcd6be.png)